setup.py

```diff
@@ -15,16 +21,42 @@ def version():
 datadir = Path(__file__).parent / 'src' / 'geocat' / 'f2py' / 'fortran'
 files = [str(p.relative_to(datadir)) for p in datadir.rglob('*.so')]
 
-setup(
-    name='geocat.f2py',
-    version=version(),
-    package_dir={
-        '': 'src',
-        'geocat': 'src/geocat',
-        'geocat.f2py': 'src/geocat/f2py',
-        'geocat.f2py.fortran': 'src/geocat/f2py/fortran',
-    },
-    package_data={'geocat.f2py.fortran': files},
-    namespace_packages=['geocat'],
-    packages=['geocat', 'geocat.f2py', 'geocat.f2py.fortran'],
-)
+setup(name='geocat.f2py',
+      version=version(),
+      maintainer='GeoCAT',
+      maintainer_email='geocat@ucar.edu',
+      python_requires='>=3.6',
+      install_requires=requirements,
+      description="""GeoCAT-f2py wraps, in Python, the compiled language
+    implementations of some of the computational functions found under the
+    GeoCAT-comp umbrella.""",
+      long_description=long_description,
+      long_description_content_type='text/markdown',
+      classifiers=[
+          'Operating System :: OS Independent',
```

Comment:
Not true, we are not OS independent for f2py, at least yet. We don't have a build process for windows.

Reply:
Great catch (of stealing from GeoCAT-comp :) ). Corrected to MacOS X and Linux now.